src/app/examples/examples.common.ts

```diff
@@ -25,6 +28,9 @@ export const COMPONENT_DECLARATIONS: any[] = [
   ChartExampleComponent,
   FontsExampleComponent,
   SpinnerExampleComponent,
+  ModalExampleComponent,
+  FirstEmbeddedModalExampleComponent,
```

Comment:
Not needed (you don't use it declarative, you use it as an entry component)
```suggestion
```

Reply:
unfortunately it won't compile without declaring it